index.bs

```diff
@@ -10699,11 +10714,13 @@ the <code>typeof</code> operator will return "function" when applied to an inter
         1.  Let &lt;|constructor|, |values|&gt; be the result
             of passing |S| and |args|.
             to the [=overload resolution algorithm=].
-        1.  Let |R| be the result of performing
-            the actions listed in the description of |constructor|
-            with |values| as the argument values.
+        1.  Let |object| be a [=new-internal|new=] object implementing |I|, with |realm| and
```

Comment:
Per my unrolling at https://github.com/heycam/webidl/pull/635#discussion_r254879231, it looks like we should use GetFunctionRealm(NewTarget), not _realm_, to match ES.

Reply:
Yeah, it looks like that's what ES does.  Given that slightly-older Firefox releases and Safari do something different (use current Realm), it's not clear to me whether the spec changed there or what.  But I agree we should just match ES.